cli_tools/common/image/importer/bootable_disk_processor.go

```diff
@@ -80,12 +78,12 @@ func newBootableDiskProcessor(request ImageImportRequest, wfPath string, logger
 	env.DaisyLogLinePrefix += "translate"
 	diskProcessor := &bootableDiskProcessor{
 		request:    request,
-		worker:     daisyutils.NewDaisyWorker(workflow, env, logger, createResourceLabeler(request)),
+		worker:     daisyutils.NewDaisyWorker(workflowProvider, env, logger, createResourceLabeler(request)),
 		logger:     logger,
 		detectedOs: detectedOs,
 		vars:       vars,
 	}
-	return diskProcessor, err
+	return diskProcessor, nil
```

Comment:
If error returned is always `nil`, remove error from `newBootableDiskProcessor` return value.

Reply:
done